share/spice/political_polls/political_polls.js

```diff
@@ -0,0 +1,67 @@
+(function (env) {
+    "use strict";
+    env.ddg_spice_political_polls = function(api_result){
+
+        // Validate the response (customize for your Spice)
+        if (!api_result || api_result.length == 0 || api_result.error) {
+            return Spice.failed('political_polls');
+        }
+        
+        if(api_result.length > 50){
+            api_result = api_result.splice(50);
+        }
+
+        // Render the response
+        DDG.require("moment.js", function() {
+            Spice.add({
+                id: "political_polls",
+
+                // Customize these properties
+                name: "Polls",
+                data: api_result,
+                meta: {
+                    sourceName: "HuffPost Pollster",
+                    sourceUrl: 'http://elections.huffingtonpost.com/pollster',
+                },
+                templates: {
```

Comment:
Please add 

```
variants: {
    tileSnippet: "large"
}
```

in here so that we'd get something like this

![screen shot 2015-09-15 at 1 19 41 pm](https://cloud.githubusercontent.com/assets/81969/9884213/92dfc01a-5bac-11e5-9759-b45d54ce0641.png)

Instead of this

![screen shot 2015-09-15 at 1 19 56 pm](https://cloud.githubusercontent.com/assets/81969/9884217/9920f818-5bac-11e5-9daa-599b00955efb.png)